addons/mrp_byproduct/models/mrp_bom.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from openerp import fields, models
+
+
+class MrpBom(models.Model):
+    _name = 'mrp.bom'
+    _description = 'Bill of Material'
+    _inherit = 'mrp.bom'
```

Comment:
_inherit and _name both are same object. no need to define _name. _inherit is enough. no need to define description also.